go/teams/loader.go

```diff
@@ -437,6 +444,15 @@ func (l *TeamLoader) load2InnerLockedRetry(ctx context.Context, arg load2ArgT) (
 	if !arg.forceFullReload {
 		// Load from cache
 		ret = l.storage.Get(ctx, arg.teamID, arg.public)
+		// as part of rollout of signchain v2.3, if a reverify is needed
+		// then treat it as a cache miss and ask for a full reload.
+		if ret != nil {
+			teamSigChainState := TeamSigChainState{inner: ret.Chain}
+			if teamSigChainState.ReverifyNeededForHighSkips() {
+				arg.forceFullReload = true
+				ret = nil
+			}
+		}
```

Comment:
Move to `load2DecideRepoll` and return `discardCache` if possible. Arg doesn't need to be modified as it's not used later.

But also.. the difference between all this cache adapting and just busting the cache. Is it just that teams with 1 sigchain link are not busted?

Reply:
def agree about moving the code and using the infrastructure that's already there.

Reply:
oh yeah forgot about that, that it's already repairing itself. Nice!